mmdet3d/ops/pointnet_modules/point_sa_module.py

```diff
@@ -110,7 +114,9 @@ def _sample_points(self, points_xyz, features, indices, target_xyz):
         Args:
             points_xyz (Tensor): (B, N, 3) xyz coordinates of the features.
             features (Tensor): (B, C, N) features of each point.
```

Comment:
torch.Tensor, optional
If `features` is a necessary input variable, please remove the default value.